### DISCUSSION TITLE
Getting error

### DISCUSSION THREAD

**Discussion by hiaeron** [medspacy/medspacy#99] [Q&A] [answered]
I am new to the MedSpacy ,I followed all the instructions to install MedSpacy, while executing the Basic usage Example I am getting the following error:

"[E001] No component 'target_matcher' found in pipeline. Available names: ['medspacy_pyrush', 'medspacy_target_matcher', 'medspacy_context']"

**Comment by abchapman93** [comment, marked as answer]
Ah yes, this was an oversight. As of the new version of medspaCy which uses spaCy 3, each pipeline component will have the prefix "medspacy_*". So in this case you should use:
`nlp.get_pipe("medspacy_target_matcher")`

Thanks for pointing this out! I'll update this in the README.

**Reply by hiaeron** [reply]
Thanks, @abchapman93 .. it worked
